### PR TITLE
feat: prefab changes to community entries

### PR DESCRIPTION
**Pull request by davidejensen**
# Pull Request Description

## What does this PR change?
This PR adds some prefabs in the community and my community entries prefabs to avoid conflicts and allow easier integration of community voice chat changes

## Test Instructions
Simple smoke test of the communities browser that validates everything looks as before in the list "My Communities" and community browser

### Additional Testing Notes
- Note any edge cases to verify
- Mention specific areas that need careful testing
- List known limitations or potential issues

## Quality Checklist
- [ ] Changes have been tested locally
- [ ] Documentation has been updated (if required)
- [ ] Performance impact has been considered
- [ ] For SDK features: Test scene is included

## Code Review Reference
Please review our [Code Review Standards](https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md) before submitting.

### CODE REVIEW

File: Explorer/Assets/DCL/Communities/CommunitiesBrowser/ListenersCountView.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+using TMPro;
+using UnityEngine;
+
+namespace DCL.Communities.CommunitiesBrowser
+{
+    public class ListenersCountView : MonoBehaviour
+    {
+        [field: SerializeField]
+        public TMP_Text ParticipantCount;
```
Comment (mihakrajnc) [inline Explorer/Assets/DCL/Communities/CommunitiesBrowser/ListenersCountView.cs:9]: Should this me a property? If not the `field:` isn't needed.

**Review by sandrade-dcl — APPROVED** [review]
